modules/ai/assets/js/editor/icons/plus-circle-icon.js

```diff
@@ -4,12 +4,12 @@ import { SvgIcon } from '@elementor/ui';
 const PlusCircleIcon = React.forwardRef( ( props, ref ) => {
 	return (
 		<SvgIcon viewBox="0 0 24 24" { ...props } ref={ ref }>
-			<g clipPath="url(#clip0_2336_467)">
+			<g clipPath="url(#clipPath)">
```

Comment:
seems like the ID should be unique on the whole page and not only in the context of this element:
https://stackoverflow.com/questions/15911717/clippath-in-multiple-svg-tags

Reply:
Removed